docs/release/release.md

```diff
@@ -0,0 +1,72 @@
+## Preparing the release
+
+1. Create a release branch in the form of `release/[DD-MM-YYYY]`
+2. Freeze the versions of each component on the requirements.yaml file to this branch. Please see the [section below](##getting-the-latest-versions-of-all-components).
```

Comment:
I think you can put the instructions in line here, instead of a different section